CHANGELOG.md

```diff
@@ -77,6 +77,10 @@
         * Add error `.invalidAPIClient` = 6
         * Add error `.cannotCastRequest`= 7
         * Add error `.jsonSerializationFailure` = 8
+  * BraintreePayPal
+    * Add `BTPaymentFlowError` case
+        * Add error `.webSessionError` = 8
+    
```

Comment:
We want to move this under the `unreleased` section since it was added in the `fpti-feature` branch and that branch hasn't been merged in or released yet.